database/migrations/1737364289031_create_about_us_generals_table.ts

```diff
@@ -0,0 +1,28 @@
+import { BaseSchema } from "@adonisjs/lucid/schema";
+
+export default class extends BaseSchema {
+  protected tableName = "about_us_general";
+
+  async up() {
+    this.schema.createTable(this.tableName, (table) => {
+      table.integer("id").primary().defaultTo(1);
+      table.text("description").notNullable();
+      table.text("cover_photo_key").notNullable();
+
+      table.timestamp("created_at");
+      table.timestamp("updated_at");
+    });
+
+    this.schema.raw(
+      `ALTER TABLE ${this.tableName} ADD CONSTRAINT single_row CHECK (id = 1);`,
+    );
+  }
+
+  async down() {
+    this.schema.raw(
+      `ALTER TABLE ${this.tableName} DROP CONSTRAINT IF EXISTS single_row;`,
+    );
+
+    this.schema.dropTable(this.tableName);
+  }
+}
```

Comment:
Dropping the constraint shouldn't be necessary since we're about to drop the table anyway